app/articles/error.tsx

```
@@ -0,0 +1,37 @@
+"use client";
+
+import { Terminal } from "lucide-react";
+import Link from "next/link";
+import { useEffect } from "react";
+
+import { Container } from "@/components/container";
+import { Alert, AlertDescription, AlertTitle } from "@/components/ui/alert";
+import { buttonVariants } from "@/components/ui/button";
+
+export default function Error({
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)